pkg/bundle/controller.go

```diff
@@ -215,3 +226,15 @@ func inNamespacePredicate(namespace string) predicate.Predicate {
 		return object.GetNamespace() == namespace
 	})
 }
+
+// labelsMatchSelector returns true is all objLabels matches the label selector
+// and false otherwise
+func labelsMatchSelector(objLabels map[string]string, selector *metav1.LabelSelector) bool {
+	labelRequirements, _ := labels.ParseToRequirements(metav1.FormatLabelSelector(selector))
```

Comment:
I think you can use `metav1.LabelSelectorAsSelector` here.

Reply:
Indeed! Good catch!